packages/trader/src/Modules/CFD/Containers/trading-password-manager.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Text, Button, Icon, MultiStep, SendEmailTemplate } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { getCFDPlatformLabel, CFD_PLATFORMS, WS } from '@deriv/shared';
+import ChangePasswordConfirmation from './cfd-change-password-confirmation.jsx';
+
+const ChangePassword = ({ platform, onConfirm }) => (
+    <div className='cfd-change-password'>
+        <Icon
+            className='cfd-change-password__icon'
+            icon={platform === CFD_PLATFORMS.MT5 ? 'IcMt5OnePassword' : 'IcDxtradeOnePassword'}
+            width='122'
+            height='108'
+        />
+        <Text as='p' align='center' size='s' weight='bold'>
+            <Localize
+                i18n_default_text='{{platform}} password'
+                values={{
+                    platform: getCFDPlatformLabel(platform),
+                }}
+            />
+        </Text>
+        <Text as='p' align='center' className='cfd-change-password__description' size='xs'>
+            {platform === CFD_PLATFORMS.MT5 ? (
+                <Localize i18n_default_text='Use this password to log in to your DMT5 accounts on the desktop, web, and mobile apps.' />
+            ) : (
+                <Localize i18n_default_text='Use this password to log in to your Deriv X accounts on the web and mobile apps.' />
+            )}
+        </Text>
+        <Button
+            className='dc-btn dc-btn--primary dc-btn__large dc-modal__container_cfd-reset-password-modal__button'
+            onClick={onConfirm}
+        >
+            <Text size='xs' weight='bold' color='colored-background'>
+                <Localize
+                    i18n_default_text='Change {{platform}} password'
+                    values={{
+                        platform: getCFDPlatformLabel(platform),
+                    }}
+                />
+            </Text>
+        </Button>
+    </div>
+);
+
+ChangePassword.propTypes = {
+    onConfirm: PropTypes.func,
+    platform: PropTypes.string,
+};
+
+const PasswordReset = ({ email, platform }) => {
+    const onClickSendEmail = React.useCallback(() => {
+        WS.verifyEmail(email, 'trading_platform_password_reset', {
+            url_parameters: {
+                platform,
+            },
+        });
+    }, [email, platform]);
+
+    React.useEffect(() => {
+        onClickSendEmail();
+    }, [onClickSendEmail]);
```

Comment:
I can't understand this behavior, why is this effect necessary?

Reply:
@reza-fs This will send a verification email when this component is mounted. Later, the user can click on `Resend email` button to send another verification email

Reply:
Makes sense now. Thanks.